api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -260,17 +275,36 @@ public void test_nonUniqueEmailPatterns() {
         .containsExactlyElementsIn(similarInstViaDb.getEmailAddresses());
   }
 
-  @Test
+  // Display Name cannot be empty
```

Comment:
That's not what this test is intended to show.  Please update the `new Institution()` to supply a valid Institution.

Reply:
Ahh i see its trying to verify updateInstitution with valid institute but a shortName thats not in DB